DFImageManager/Source/Core/Utilities/DFProxyImageManager.m

```diff
@@ -83,6 +83,11 @@ - (BOOL)canHandleRequest:(nonnull DFImageRequest *)request {
     return [_manager canHandleRequest:_DF_TRANSFORMED_REQUEST(request)];
 }
 
+- (BOOL)hasCacheWithRequest:(nonnull DFImageRequest *)request {
+    NSParameterAssert(request);
+    return [_manager hasCacheWithRequest:request];
```

Comment:
Proxy is used to transform requests before they get to the actual image manager. We use it to make `DFImageManager` work with our custom entities. So it should be `[_manager hasCacheWithRequest:_DF_TRANSFORMED_REQUEST(request)]`